packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Checkmarks.Example.tsx

```diff
@@ -25,198 +24,188 @@ const keys: string[] = [
-                    canCheck: true,
-                    isChecked: selection![keys[10]],
-                    onClick: this._onToggleSelect
-                  }
-                ]
-              },
-              text: 'Split Button',
-              canCheck: true,
-              isChecked: selection![keys[8]],
-              split: true,
-              onClick: this._onToggleSelect,
-              disabled: true
-            },
-            {
-              key: keys[11],
-              iconProps: {
-                iconName: 'MusicInCollectionFill'
-              },
-              subMenuProps: {
-                directionalHint: DirectionalHint.leftCenter,
-                items: [
-                  {
-                    key: keys[12],
-                    text: 'Email message',
-                    canCheck: true,
-                    isChecked: selection![keys[12]],
-                    onClick: this._onToggleSelect
-                  },
-                  {
-                    key: keys[13],
-                    text: 'Calendar event',
-                    canCheck: true,
-                    isChecked: selection![keys[13]],
-                    onClick: this._onToggleSelect
-                  }
-                ]
-              },
-              text: 'Split Button Left Menu',
-              canCheck: true,
-              isChecked: selection![keys[11]],
-              split: true,
-              onClick: this._onToggleSelect
-            },
-            {
-              key: keys[12],
-              iconProps: {
-                iconName: 'MusicInCollectionFill'
-              },
-              subMenuProps: {
-                items: [
-                  {
-                    key: keys[12],
-                    name: 'Email message',
-                    canCheck: true,
-                    isChecked: selection![keys[12]],
-                    onClick: this._onToggleSelect
-                  }
-                ]
-              },
-              name: 'Split Button Disabled Primary',
-              split: true,
-              primaryDisabled: true
-            },
-            {
-              key: keys[13],
-              iconProps: {
-                iconName: selection![keys[13]] ? 'SingleBookmarkSolid' : 'SingleBookmark'
-              },
-              name: selection![keys[13]] ? 'Toogled command no checkmark' : 'Toogle command no checkmark',
-              canCheck: false,
-              isChecked: selection![keys[13]],
-              onClick: this._onToggleSelect
-            }
-          ]
-        }}
-      />
-    );
-  }
-
-  private _onToggleSelect(ev?: React.MouseEvent<HTMLButtonElement>, item?: IContextualMenuItem): void {
-    const { selection } = this.state;
-    ev!.preventDefault();
-    selection![item!.key] = !selection![item!.key];
-
-    this.setState({
-      selection: selection
-    });
-  }
-}
+  return (
+    <DefaultButton
+      text="Click for ContextualMenu"
+      menuProps={{
```

Comment:
If we're being super pedantic (which it sounded like we wanted to in examples last time it was discussed), this shouldn't be defined inline either--basically **no** inline object or array definitions, since even though it's not likely to cause problems in examples, we want to demonstrate good patterns that real apps should use. Instead define above with `useMemo(() => /*obj*/, [menuItems])`